steps/virtual-device-testing-for-android/0.9.2/step.yml

```diff
@@ -0,0 +1,276 @@
+title: '[PRIVATE-BETA] Virtual Device Testing for Android'
+summary: Run Android UI tests on virtual devices
+description: |-
+  Run Android UI tests on virtual devices. Available test types are instrumentation, robo, gameloop.
+  The minimal setup of this step would be to select test type. If you selected instrumentation, don't forget to set __Test APK path__
+  under the __Instrumentation Test__ group as well.
+website: https://github.com/bitrise-steplib/steps-virtual-device-testing-for-android
+source_code_url: https://github.com/bitrise-steplib/steps-virtual-device-testing-for-android
+support_url: https://github.com/bitrise-steplib/steps-virtual-device-testing-for-android/issues
+published_at: 2017-08-29T14:25:03.290376793+02:00
+source:
+  git: https://github.com/bitrise-steplib/steps-virtual-device-testing-for-android.git
+  commit: 7f399837084c2032d21315453e11a4c4abbe6767
+host_os_tags:
+- ubuntu
+project_type_tags:
+- android
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-virtual-device-testing-for-android
+deps:
+  brew:
+  - name: go
+  apt_get:
+  - name: golang
+    bin_name: go
+is_always_run: false
+is_skippable: false
+inputs:
+- apk_path: $BITRISE_APK_PATH
+  opts:
+    description: |
+      The path to the APK you want the tests run with. By default `gradle-runner` step exports `BITRISE_APK_PATH` env, so wou won't need to change this input.
+    is_required: true
+    summary: |
+      The path to the APK you want the tests run with. By default `gradle-runner` step exports `BITRISE_APK_PATH` env, so wou won't need to change this input.
```

Comment:
remove: `By default `gradle-runner` step exports `BITRISE_APK_PATH` env, so wou won't need to change this input.`, it is enough to mention in the description